CONTRIBUTING.md

```diff
@@ -627,6 +631,74 @@ required.
 Essentially, the steps you have to take to deploy manually are the same that
 `skaffold` will perform, and they're outlined at the previous section.
 
+#### Kubernetes E2E tests
+
+Kubernetes integration is quite complicated, a lot of things can go wrong.
+
+To cope with the complexity and ensure we maintain high quality, we use
+E2E (end-to-end) tests.
+
+> E2E tests normally run at CI, so there's typically no need to run them
+> manually.
+
+##### Requirements
+
+* `kubernetes` cluster (`minikube` has special support, but any cluster should
+  work)
+* `docker`
+* `kubectl`
+* `bash`
```

Comment:
So these tests can't work from a Windows machine?

Reply:
Technically, you should be able to run tests on Windows via git bash. I didn't test it though...
Also, I want to note that running tests *on* Windows doesn't correlate to testing *Windows clusters*, and that we don't address that yet.

Reply:
Yeah I have bash through busybox so I think this might work. I think this is fine for now. We'll need to support windows fully at some point though. We do support users developing on windows.

Reply:
> We do support users developing on windows.
Yeah, that's a bit tricky to do for k8s, but we'll get there.

The dev flow (the one that uses skaffold) *is* currently limited to linux (on the contrary to e2e tests, what can be launched from Linux/macOS/Windows in theory). We can add support for using Windows/macOS there if we cross-compile `vector` for linux - but that would significantly increas the build time and won't be fit for quick iterations anymore.
I want to address it at some point. But not now - and most likely after we ship the "MVP for the integration".

Reply:
Tried it on Windows - got stuck at the building vector into release artifacts. This should be easy to fix.
We just need to give Windows a little bit more attention to solve all the necessary dependencies before thinking about k8s-specific stuff.